pkg/route/OWNERS

```diff
@@ -2,15 +2,11 @@ approvers:
   - frobware
   - knobunc
   - Miciah
-  - sgreene570
-  - smarterclayton
```

Comment:
You could move these to `emeritus_approvers` per <https://www.kubernetes.dev/docs/guide/owners/#cleanup>.

Reply:
Clayton added to the emeritus approvers for his great contribution to all things OpenShift. The others are nice guys but I don't see a reason to keep them in OWNERS.